final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +74,86 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster {
+  "name" => "Aries",
+  "smell" => "dog",
+  "weight" => 20,
+  "citiesDestroyed" => ["Denver", "New York", "Chicago"],
+  "luckyNumbers" => [1,23,43],
+
+  address {
```

Comment:
See how address has a different syntax highlighting than your other keys? You are missing some syntax to make sure `address` is a key just like your other ones.... basically, it should look like all your other keys above, but just have a hash as its value.